CHANGELOG.md

```diff
@@ -7,6 +7,159 @@ documentation before upgrading to a new release.
 
 Released closed milestones can be found on [GitHub](https://github.com/Icinga/icinga2/milestones?state=closed).
 
+## 2.15.0 (2025-06-18)
+
+This Icinga 2 release is focused on adding Icinga 2 dependencies support to Icinga DB, but also includes a number
+of bugfixes, enhancements and code quality improvements. Below is a summary of the most important changes, for the
+complete list of issues and PRs, please see the [milestone on GitHub](https://github.com/Icinga/icinga2/milestone/99?closed=1).
```

Comment:
I prefer the old link format from v2.14.0. You can filter, sort and it's uniform with...